Expensify/App
# Update generic bank icon for connected accounts

### Details


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/180970

### Tests / QA
1. With a brand-new account, create a new workspace on new.e
2. Go to the workspace and click "Get started"
2. Add an OPEN verified bank account following [SO step 3](https://stackoverflow.com/c/expensify/questions/342).
2. Go to the list of connected payment methods on the workspace
3. Verify the new icon is in place

### Tested On


### Screenshots


#### Web

<img width="373" alt="Screen Shot 2021-10-14 at 3 22 42 PM" src="https://user-images.githubusercontent.com/24466196/137403832-3eeebfe8-7830-4b08-a2ef-e910ef848725.png">


#### Mobile Web


#### Desktop


#### iOS


#### Android
